### PR TITLE
Add Uiconf filter to search partner in AC

### PR DESCRIPTION
**Pull request by Galmoss**
SearchByUiconf

### CODE REVIEW

**Comment by Galmoss** [comment]
FYI @tan-tan-kanarek

File: admin_console/controllers/PartnerController.php [hunk 1]
```diff
@@ -579,6 +579,22 @@ private function getPartnerFilterFromRequest(Zend_Controller_Request_Abstract $r
...
+		
+		    try {
+		        $uiConf = $adminConsolePlugin->uiConfAdmin->get($filterInput);
+		        /* @var $uiConf Kaltura_Client_Type_UIConf  */
+		        $filterInput = $uiConf->partnerId;
+		    }
+		    catch(Exception $ex) {
+		        $filterInput = "-99";
+		    }
+		    $filterType = 'byid';
```
Comment (tan-tan-kanarek) [inline admin_console/controllers/PartnerController.php]: You should keep the filter type and input for the UI, just add the partner id to $filter->idIn instead.

Comment (tan-tan-kanarek) [inline admin_console/controllers/PartnerController.php]: Apply it also to search by entry id.

Comment (Galmoss) [inline admin_console/controllers/PartnerController.php]: @tan-tan-kanarek, done.

**Comment by tan-tan-kanarek** [comment]
Looks good.

**Pull request closed without merging**